x-pack/plugins/transform/public/app/sections/transform_management/transform_management_section.tsx

```diff
@@ -77,6 +79,23 @@ export const TransformManagement: FC = () => {
   // Call useRefreshInterval() after the subscription above is set up.
   useRefreshInterval(setBlockRefresh);
 
+  const unauthorizedTransformsWarning = useMemo(() => {
+    const unauthorizedCnt = transforms.filter((t) => needsReauthorization(t)).length;
+    return unauthorizedCnt > 0 ? (
+      <>
+        <EuiCallOut
+          color="warning"
+          title={i18n.translate('xpack.transform.transformList.unauthorizedTransformsCallout', {
+            defaultMessage:
+              '{unauthorizedCnt, plural, one {A transform was installed but requires more permissions to run.} other {# transforms were installed but require more permissions to run.}} Contact your administrator to request the required privileges.',
```

Comment:
We can use the same string as above if the user has correct permissions: `A transform was installed by a user who did not have the permissions required to run it. Reauthorize it in the Actions menu.`

In the case of the user without proper permissions, I propose to use the following: `This transform was created with insufficient permissions. Contact your administrator to request the required permissions.` So we can use a single term instead of permissions and privileges.

Reply:
Updated here [`1775dd0` (#154736)](https://github.com/elastic/kibana/pull/154736/commits/1775dd00625b2662d49343667c34413abd128e82)